libs/openFrameworks/sound/ofRtAudioSoundStream.h

```diff
@@ -9,6 +9,18 @@
 class RtAudio;
 typedef unsigned int RtAudioStreamStatus;
 
+enum ofRtAPI {
```

Comment:
I tried that but RtAudio::Api is not accessable unless you include rtAudio.h, otherwise it complains about an incomplete type